frontend/src/features/documents/list/DocumentListView.tsx

```diff
@@ -59,6 +60,17 @@ export const DocumentListView: React.FunctionComponent<IDocumentListViewProps> =
     return [];
   }, [documentResults, sort, filters]);
 
+  const [isDetailsVisible, setIsDetailsVisible] = useState(false);
+  const [selectedDocument, setSelectedDocument] = useState<Api_Document | undefined>(undefined);
+
+  const handleViewDetails = (values: Api_Document) => {
+    setIsDetailsVisible(true);
+    setSelectedDocument(values);
+  };
+  const handleDelete = (values: Api_Document) => {
+    // Todo: need to have the parent entity in context
```

Comment:
At this point there is no activity/file to make the delete functionality work

Reply:
How about the ** entityId: number** that is passed in as props? I believe that would be the parent entity

Reply:
The way we did it with notes is create a "placeholder" activity on our local databases and have the Test page use that ID to create/update/delete notes from it. See example in TestNotes.tsx page - I believe a similar pattern could be followed here while we wait for Activity forms to be implemented...